src/test/java/me/magnet/consultant/ConsultantConfigurationProviderTest.java

```diff
@@ -47,6 +49,7 @@ public void tearDown() {
 	public void verifyLoadingStringProperty() throws Exception {
 		createConsultant(ImmutableMap.of("config/oauth/some.key", "some-value"));
 		ConsultantConfigurationProvider provider = new ConsultantConfigurationProvider(consultant);
+		Assert.assertThat(provider.has(CONFIGURATION_KEY), is(true));
```

Comment:
Also make a separate method for this test. This is interweaving two different tests.